app/Policies/TicketPolicy.php

```diff
@@ -35,7 +35,7 @@ public function view(User $user, Ticket $ticket): bool
 
     public function create(User $user): bool
     {
-        if ($user->created_at->diffInDays() < 1 || $user->is_muted || $user->banned_at) {
+        if ($user->created_at->diffInDays(now(), true) < 1 || $user->is_muted || $user->banned_at) {
```

Comment:
Should this be `Carbon::now()` or is the function overloaded to accept a `time_t`?

Also applies to next two files.

Reply:
No harm in using `Carbon::now()`. I've also switched some of these to instead use `->diffInHours()`. Updated in latest.